src/test/java/com/snowflake/kafka/connector/SnowflakeSinkTaskForStreamingIT.java

```diff
@@ -41,109 +41,112 @@
  * com.snowflake.kafka.connector.internal.streaming.SnowflakeSinkServiceV2}
  */
 public class SnowflakeSinkTaskForStreamingIT {
-
+  private int taskId;
+  private int partitionCount;
   private String topicName;
-  private static int partition = 0;
-  private TopicPartition topicPartition;
   private Map<String, String> config;
-
-  @Mock Logger logger = Mockito.mock(Logger.class);
-
-  @InjectMocks @Spy
-  private LoggerHandler loggerHandler = Mockito.spy(new LoggerHandler(this.getClass().getName()));
-
-  @InjectMocks private SnowflakeSinkTask sinkTask1 = new SnowflakeSinkTask();
-
+  private List<TopicPartition> topicPartitions;
+  private SnowflakeSinkTask sinkTask;
+  private InMemorySinkTaskContext sinkTaskContext;
+  private List<SinkRecord> records;
+  private Map<TopicPartition, OffsetAndMetadata> offsetMap;
+
+  // sets up default objects required for sunny day tests
+  // NOTE: topicPartitions, records and corresponding offsetmap have a count of one each
   @Before
-  public void setup() {
-    topicName = TestUtils.randomTableName();
-    topicPartition = new TopicPartition(topicName, partition);
-
-    // config
-    this.config = TestUtils.getConfForStreaming();
-    this.config.put(BUFFER_COUNT_RECORDS, "1"); // override
-    this.config.put(INGESTION_METHOD_OPT, IngestionMethodConfig.SNOWPIPE_STREAMING.toString());
-    SnowflakeSinkConnectorConfig.setDefaultValues(this.config);
+  public void setup() throws Exception {
```

Comment:
goal of this method is to move a lot of this setup outside of the test itself. makes creating a new test a litttle bit harder in that you'd need to read this setup (but you could just create your own variables instead of using this). I think this cleans it up a lot, and makes the int test more readable

Reply:
moved to a following pr, so this comment is out of context